projects/igniteui-angular/src/lib/toast/toast.component.ts

```diff
@@ -45,95 +45,185 @@ let NEXT_ID = 0;
+     * Default value is `true`.
+     * ```html
+     * <igx-toast [autoHide] = "false"></igx-toast>
+     * ```
+     * ```typescript
+     * let autoHide = this.toast.autoHide;
+     * ```
+     * @memberof IgxToastComponent
      */
     @Input()
     public autoHide = true;
 
     /**
-     * The duration of time span in ms which the IgxToast component will be visible
+     * Sets/gets the duration of time span(in milliseconds) which the toast will be visible
      * after it is being shown.
-     * Default value is 4000
+     * Default value is `4000`.
+     * ```html
+     * <igx-toast [displayTime] = "2500"></igx-toast>
+     * ```
+     * ```typescript
+     * let displayTime = this.toast.displayTime;
+     * ```
+     * @memberof IgxToastComponent
      */
     @Input()
     public displayTime = 4000;
 
     /**
-     * The IgxToast component visual state state
+     * Enables/Disables the visibility of the toast.
+     * If not set, the `isVisible` attribute will have value `false`.
+     * ```html
+     * <igx-toast [isVisible] = "true"></igx-toast>
+     * ```
+     * ```typescript
+     * let isVisible = this.toast.isVisible;
+     * ```
+     * @memberof IgxToastComponent
      */
     @Input()
     public isVisible = false;
 
     /**
-     * The message that will be shown message by the IgxToast component
+     * Sets/gets the message that will be shown by the toast.
+     * ```html
+     * <igx-toast [message] = "Notification"></igx-toast>
+     * ```
+     * ```typescript
+     * let toastMessage = this.toast.message;
+     * ```
+     * @memberof IgxToastComponent
      */
     @Input()
     public message: string;
 
     /**
-     * Specifies the position of the IgxToast component. Possible options are IgxToastPosition.Top,
-     * IgxToastPosition.Middle, IgxToastPosition.Bottom
+     * Sets/gets the position of the toast.
+     * If not set, the `position` attribute will have value `IgxToastPosition.Bottom`.
+     * ```html
+     * <igx-toast [position] = "top"></igx-toast>
+     * ```
+     * ```typescript
+     * let toastPosition = this.toast.position;
+     * ```
+     * @memberof IgxToastComponent
      */
     @Input()
     public position: IgxToastPosition = IgxToastPosition.Bottom;
 
     /**
-     * Return the nativeElement of the component
+     * Gets the nativeElement of the toast.
+     * ```typescript
+     * let nativeElement = this.toast.element;
+     * ```
+     * @memberof IgxToastComponent
      */
     public get element() {
         return this.elementRef.nativeElement;
     }
-
+    /**
+     *@hidden
+     */
     private timeoutId;
 
     constructor(
         private elementRef: ElementRef,
         @Optional() private navService: IgxNavigationService) { }
 
     /**
-     * Shows the IgxToast component and hides it after some time span
-     * if autoHide is enabled
+     * Shows the toast and hides it after the `displayTime` is over
```

Comment:
Shows the toast. 
If `autoHide` is enabled the toast will hide after `displayTime` is over.